v2/charts/azure-service-operator/values.yaml

```diff
@@ -86,14 +86,19 @@ webhook:
 # repository).
 image:
   repository: mcr.microsoft.com/k8s/azureserviceoperator:v2.6.0
-  kubeRBACProxy: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
 
 # 'metrics' define settings for the metrics from controller.
 # 'address' field defines the metrics binding address on which metrics
 metrics:
   enable: true
+  # secure controls whether metrics should be served via 'http' or 'https'.
+  # Flagging secure as 'true' would use https
+  # Refer to https://azure.github.io/azure-service-operator/guide/metrics/ for more information
+  secure: true
+  # profiling endpoints are only enabled when serving metrics securely
```

Comment:
minor: Say what profiling is (it enables /pprof endpoints), in addition to its restrictions. Though, see below about this restriction because I am not sure it is needed?